env/dev/sim_no-threads.yaml

```diff
@@ -0,0 +1,19 @@
+spack:
+  definitions:
+    - when: platform == 'linux'
+      root: [root@6.18.04+fortran+gdml+http+pythia6+pythia8+vc~vdt~python~tmva]
+    - when: platform == 'darwin'
+      root: [root@6.18.04+fortran+gdml+http+pythia6+pythia8+vc~vdt~python~tmva+aqua]
+  specs:
+    - pcre+jit
+    - pythia6@428-alice1
+    - hepmc@2.06.09 length=CM momentum=GEV
+    - pythia8@8301
+    - geant4@10.05.p01~qt~vecgeom~opengl~x11~motif~data~clhep+threads
```

Comment:
Just noticed:
The env is called `no-threads`, but here we have `+threads`.

Reply:
good catch, thats a copy&paste fail when I reverted the Geant4 bump, will correct